ha/core/system_health/system_health.py

```diff
@@ -393,7 +394,7 @@ def process_event(self, healthevent: HealthEvent):
                                         cluster_id=healthevent.cluster_id, site_id=healthevent.site_id,
                                         rack_id=healthevent.rack_id, storageset_id=healthevent.storageset_id,
                                         node_id=healthevent.node_id, server_id=healthevent.node_id,
-                                        storage_id=healthevent.node_id)
+                                        storage_id=healthevent.node_id, cvg_id=self.cvg_id)
```

Comment:
same as above

Reply:
I think this already checked-in so, you can leave it

Reply:
Rebased.